docs/pages/enterprise/sso.mdx

```diff
@@ -4,30 +4,32 @@ description: How to set up single sign-on (SSO) for SSH using Gravitational Tele
 h1: Single Sign-On (SSO) for SSH
 ---
 
-Users of the Enterprise edition of Teleport can login with SSH, Kubernetes, Databases and Web applications
-through a Single Sign-On (SSO) provider used the rest of the organization.
+The Enterprise edition of Teleport, self-hosted or Teleport Cloud, allows for using a organization's
```

Comment:
I think it would be clearer to put the edition compatibility statement after the introductory paragraph, e.g.,

```markdown
Teleport enables you to use an organization's Single Sign-On (SSO)
provider to authenticate and authorize access for SSH, Kubernetes,
databases, desktops, and web applications. 

Teleport Enterprise and Teleport Cloud support the OIDC and SAML protocols
for SSO providers. This is in addition to the [GitHub SSO](../setup/admin/github-sso.mdx)
support available to the open source edition of Teleport.
```

This way, we can emphasize the description of the feature separately from the compatibility statement.